README.md

```diff
@@ -66,7 +66,7 @@ Your new project consists of:
 - Builds for zkVM program, the publisher app, and any other Rust code.
 
   ```sh
-  cargo build
+  cargo build --release
```

Comment:
Is the only reason we _must_ run this _first_ is because forge is looking to get an Elf.sol that this builds?

Or should we be optimizing here for the `forge b` next?

Reply:
Indeed `cargo build` generates the `ImageID.sol` and `Elf.sol` files, which is the primary reason it needs to run first. I don't know of any way to achieve this just by running `forge build` or `forge test`.

I don't see any reason to run with `--release`, since the performance is not an issue in client-server mode.

Reply:
Also `--release` takes a lot longer to build, so if you're in client/server, it might not be worth it.